ironfish-cli/src/commands/wallet/multisig/dkg/create.ts

```diff
@@ -0,0 +1,235 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { RPC_ERROR_CODES, RpcClient, RpcRequestError } from '@ironfish/sdk'
+import { Flags } from '@oclif/core'
+import { IronfishCommand } from '../../../../command'
+import { RemoteFlags } from '../../../../flags'
+import * as ui from '../../../../ui'
+
+export class DkgCreateCommand extends IronfishCommand {
+  static description = 'Interactive command to create a multisignature account using DKG'
+
+  static flags = {
+    ...RemoteFlags,
+    participantName: Flags.string({
+      char: 'n',
+      description: 'The name of the participant to use for DKG',
+    }),
+    accountName: Flags.string({
+      char: 'a',
+      description: 'The name to set for the imported account',
+    }),
+  }
+
+  async start(): Promise<void> {
+    const { flags } = await this.parse(DkgCreateCommand)
+    const client = await this.connectRpc()
+    await ui.checkWalletUnlocked(client)
+
+    const participantName = flags.participantName
+    const accountName = flags.accountName
```

Comment:
i think ideally these names are the same: it can lead to some confusion to have an identity name that doesn't match the account name. should we have a singler `name` flag?